recipes/meta/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "meta" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "a35dd5bc3ccffaaf83aa488233467ae85dc5374aea3068c34a6dc85ea29e27a9" %}
+{% set build_variant = "osmesa" if linux else "qt" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bagari/meta/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage('meta', max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - cmake
+    - make  # [not win]
+
+  host:
+    - python >=3.9,<3.10
+    - pip
+    - boost =1.82
+    - itk =5.3.0
+    - libitk =5.3.0
+    - libitk-devel =5.3.0
+    - hdf5 >=1.14.2,<1.14.3.0a0
```

Comment:
Conda-forge generally don't pin so hard on these dependencies and rather allows global pinnings to take over.

We prefer:
* Global pinnings + light patching

in opposition to:
* overly specific pins

to ensure broader compatibility.

Would it be possible to remove these pins?

Reply:
Thank you for your suggestions. I updated the `meta.yaml` and tried to make it more general.